content/docs/1.3.3/volumes-and-nodes/maintenance.md

```diff
@@ -1,59 +1,52 @@
 ---
-title: Node Maintenance Guide
+title: Node Maintenance and Kubernetes Upgrade Guide
 weight: 6
 ---
 
-This section describes how to handle planned maintenance of nodes.
+This section describes how to handle planned node maintenance or upgrading Kubernetes version for the cluster.
 
 - [Updating Node OS or Container Runtime](#updating-the-node-os-or-container-runtime)
-- [Updating Kubernetes](#updating-kubernetes)
 - [Removing a Disk](#removing-a-disk)
 - [Removing a Node](#removing-a-node)
+- [Updating Kubernetes](#upgrading-kubernetes)
 
 ## Updating the Node OS or Container Runtime
 
 1. Cordon the node. Longhorn will automatically disable the node scheduling when a Kubernetes node is cordoned.
 
 1. Drain the node to move the workload to somewhere else.
 
-    You will need to use `--ignore-daemonsets` and `--pod-selector='app!=csi-attacher,app!=csi-provisioner,app!=longhorn-admission-webhook,app!=longhorn-conversion-webhook,app!=longhorn-driver-deployer'` options to drain the node.
-    The `--ignore-daemonsets` is needed because Longhorn deployed some daemonsets such as `Longhorn manager`, `Longhorn CSI plugin`, `engine image`.
-    The `--pod-selector='app!=csi-attacher,app!=csi-provisioner,app!=longhorn-admission-webhook,app!=longhorn-conversion-webhook,app!=longhorn-driver-deployer'` is needed so that Longhorn can properly detaches Longhorn volumes (see the [GitHub issue](https://github.com/longhorn/longhorn/issues/3304) for more detail).
+   You will need to use `--ignore-daemonsets` and `--force` to drain the node.
+   The `--ignore-daemonsets` is needed because Longhorn deployed some daemonsets such as `Longhorn manager`, `Longhorn CSI plugin`, `engine image`.
+   The `--force` is needed because some Longhorn pods (instance manager and share manager pods) are not managed by a ReplicationController, Job, or DaemonSet.
```

Comment:
Do you mean right now without `--force` the drain will fail?

Reply:
Yes, without --force the drain will fail

Reply:
cc @longhorn/qa

Reply:
We have tested v1.3.3 drain without `force`, it seemed to have worked. https://longhorn.github.io/longhorn-tests/manual/pre-release/node/node-drain-deletion/.  We have also tested drain using Rancher UI, it seems without selecting `force` option, we were able to drain successfully.

Reply:
@PhanLe1010 had a good comment for the context.

[/datastore/longhorn.go#L2775-L2777](https://github.com/longhorn/longhorn-manager/blob/b1b4fc0515ac34c43601c3c6503ac603e6ba9b54/datastore/longhorn.go#L2775-L2777)
```go
			// This field is needed so that `kubectl drain` can work without --force flag
			// See https://github.com/longhorn/longhorn/issues/1286#issuecomment-623283028 for more details
			Controller:         &controller,
```